tests/ckeditor/ckeditor.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect */
+
+'use strict';
+
+bender.amd.require( 'ckeditor', 'ckeditor-core' );
+
+describe( 'getPluginPath()', function( CKEDITOR, core ) {
```

Comment:
> this means we have to keep in mind the order of required modules and write them down in every describe's callback

Hm... Makes sense. Usually we're testing one thing in one file, so there will be one `describe()` call. But eventually this mechanism will be adopted by benderjs-amd and then other projects will start using it and may want to use far more `describe()`s.

Besides that, I'm ok with both APIs.